tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -4333,7 +4333,43 @@ HRESULT STDMETHODCALLTYPE CorProfiler::JITCachedFunctionSearchStarted(FunctionID
         return S_OK;
     }
 
-    *pbUseCachedFunction = true;
```

Comment:
Shouldn't this still be here, for the .NET FX case? 🤔

Reply:
yes :nod-yes:

Reply:
Reinstated it [here](https://github.com/DataDog/dd-trace-dotnet/pull/6184/commits/3aa6bc777ea7d4d6c4f9ec6613e88e45b2af0e31)